command assembler

F1nx opened the pull request:
Add an `HTML` _"command assembler"_ for an easy-to-use GUI for those who are new to `asemica`.

- Works on major browsers.
- Currently incompatible with mobile platform(s), but it would be unlikely that users would use this on their phone, anyhow.

Open and subject to change! Please review the style, usability, and functionality.

Commit:
Add files via upload

Commit:
mention assembler.html

ForgivenNin commented:
The main thing that I would like to clarify before or if this merges is that sometimes to invoke `asemica`, you may have to use other ways than `./asemica`. As someone who is using a Mac, I have to `cd` the folder that contains it, and add `perl asemica` before putting in the arguments.

Commit:
add corpus file (#8) (#1)

Supplies a corpus file for use by default with the program (did not modify asemica).
Modified text for processing.

Donated by Kassidy Jones (2017).
See README.md line 177.

Commit:
Merge branch 'master' into assembler

ghost commented:
Probably want to just move this pr into a different branch from `master`.
